osu.Game.Rulesets.Catch/Skinning/Default/PulpFormation.cs

```diff
@@ -34,11 +33,23 @@ protected PulpFormation()
             distance * MathF.Sin(angle * MathF.PI / 180),
             distance * MathF.Cos(angle * MathF.PI / 180));
 
-        [BackgroundDependencyLoader]
-        private void load(DrawableHitObject drawableObject)
+        protected void Clear()
+        {
+            for (; numPulps > 0; numPulps--)
+                InternalChildren[numPulps - 1].Alpha = 0;
```

Comment:
This is also way too cleverly written for my liking. I assume the intention here is to set `Alpha` and roll back `numPulps` to 0 at the same time, but I'd honestly just prefer a plain `for` loop and value reset to 0 at the end.